pkg/apis/mysql/v1alpha1/types.go

```diff
@@ -66,6 +66,9 @@ type ClusterSpec struct {
 	SSLSecret *corev1.LocalObjectReference `json:"sslSecret,omitempty"`
 	// SecurityContext holds pod-level security attributes and common container settings.
 	SecurityContext *corev1.PodSecurityContext `json:"securityContext,omitempty"`
+	// Tolerations allows specifying a list of tolerations for controlling which
+	// set of nodes a pod can be scheduled on
```

Comment:
nit (not worth blocking merge): `s/nodes/Nodes/` `s/pod/Pod/` and a full-stop.

Reply:
Ah sorry! I'll fix it up in a follow-up PR.